README.md

```diff
@@ -1,2 +1,78 @@
-# rbac-api-utils
-A library for multi cluster rbac api
+# RBAC API Utils
+[![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+
+A library for multi-cluster RBAC API. Access control in OCM is based on RBAC contructs  provided by Kubernetes,
+this library provides utiilies and  wrappers around the kubernetes RBAC API for convenience and 
+to achieve finer grained access control.
+
+Go to the [Contributing guide](CONTRIBUTING.md) to learn how to get involved.
+
+- The `rbac-api-utils` repository is part of the `open-cluster-management`
+  community. For more information, visit:
+  [open-cluster-management.io](https://open-cluster-management.io).
+
+## Usage
+
+To get started, create an instance of the AccessReviewer as below:
+
+```go
+
+import (
+  "github.com/stolostron/rbac-api-utils/rbac"
+)
+
+// Create an instance of AccessReviewer with the KubeConfig for the target cluster
+accessReviewer := rbac.NewAccessReviewer(myTargetKubeConfig, nil)
+
+```
+
+See [here](./pkg/rbac/rbac.go/#L49) for more information on the parameters for creation of a AccessReviewer
+
+### Supported API
+
+**GetMetricsAccess** returns the  managedclusters and namespaces on the managedclusters for which the user has access to view observability metrics. See [here](./pkg/rbac/rbac.go/#L121) for details on the input parameters and results.
```

Comment:
I think we would want to use the English term of `managed clusters` through this section since you aren't talking about the Kubernetes API objects.